packages/config/src/api/site_info.js

```diff
@@ -71,3 +82,24 @@ const getAddons = async function (api, siteId) {
     throwUserError(`Failed retrieving addons for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
+
+const getIntegrations = async function (api, ownerType, ownerId) {
+  if (ownerId === undefined) {
+    return []
+  }
+
+  try {
+    const token = api.accessToken()
+    const response = await fetch(`https://api.netlifysdk.com/${ownerType}/${ownerId}/integrations`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+
+    const integrations = await response.json()
+    return Array.isArray(integrations) ? integrations : []
+  } catch (error) {
+    // for now, we'll just ignore errors, as this is early days
```

Comment:
Do you want to still pipe them to our system logger, perhaps?

Reply:
yeah good shout, as we've got the feature flag nothing to lose here